spec/src/main/asciidoc/providers.asciidoc

```diff
@@ -61,14 +61,50 @@ The `toThrowable(Response)` method actually does the conversion work.  This meth
 
 Any methods that read the response body as a stream must ensure that they reset the stream.
 
-==== Built in ResponseExceptionMapper
+== Provider Declaration
+
+In addition to defining providers via the client definition, interfaces may use the `@RegisterProviders` annotation to define classes to be registered as providers in addition to providers registered via the `RestClientBuilder`
+
+== Automatic Provider Registration
+
+Implementations may provide any number of providers registered automatically, but t
```

Comment:
Unfinished sentence?

Reply:
Huh, weird.  I could swear I finished that. :-)